Neos.ContentRepository.Core/Classes/SharedModel/Node/NodeAggregateId.php

```diff
@@ -40,6 +40,11 @@ private function __construct(
         }
     }
 
+    public static function hasValidFormat(string $value): bool
```

Comment:
we need a unique naming thing here ... in other places we call such method `isMatchedByPattern` or have a `tryFrom`. Idk what i prefer :O buuut being consistent if we can is important imo.

Reply:
Yep, I'll happily rename the method. `tryFrom` isn't too bad actually and in sync with backed enums

Reply:
@mhsdesign we use `tryFromString` in `PropertyType` and `AbsoluteNodePath` so I went for _that_ and IMO the usage looks fine: https://github.com/neos/neos-ui/pull/3789/files#diff-546ee5592dc99c39f62a68190cc378c15d450f7fb75f94993b8a5d07f97675e7R83-R85